src/go/rpk/pkg/utils/args.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package utils
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// Wrapping cobra's matchAll function and return an error including usage String.
+// MatchAll allows combining several PositionalArgs to work in concert.
+func MatchAll(checks ...cobra.PositionalArgs) cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		for _, check := range checks {
+			if err := check(cmd, args); err != nil {
+				return fmt.Errorf("%v\n\n%v", err, cmd.UsageString())
+			}
+		}
+		return nil
+	}
+}
```

Comment:
Missing new line here :) Not sure why the linter didn't catch it ^

Reply:
Corrected, thanks.

Reply:
It's still there 😉

Reply:
Sorry for that, I had issues with my IDE.

Reply:
@0x5d yeah, since I was going to remove it in the last commit I didn’t correct it. Now the commit history doesn’t have the args.go file 👍🏽